iotc-generic-c-sdk/azure-sdk-impl/src/azure_sdk_client.c

```diff
@@ -202,7 +202,7 @@ int iotc_device_client_send_message_with_mt(const char *message, int mt )
         Map_AddOrUpdate(propMap, "mt", propText3);//
 
         if (IoTHubDeviceClient_LL_SendEventAsync(device_ll_handle, message_handle, send_confirm_callback, (void*)message_handle) !=
-            IOTHUB_MESSAGE_OK) {
+            IOTHUB_CLIENT_OK) {
```

Comment:
@sw-iot-neeravparasar can you please chack for the appropriate enumeration based om what IoTHubDeviceClient_LL_SendEventAsync() needs to return.